modules/ols-creating-lightspeed-custom-resource-file-using-cli.adoc

```diff
@@ -40,10 +40,57 @@ spec:
     defaultProvider: myOpenai
 ----
 +
-[NOTE]
-====
-For OpenShift AI vLLM use the same configuration as OpenAI but update the URL to point to your Virtual Large Language Model (vLLM) endpoint. If OpenShift Lightspeed operates in the same cluster as the vLLM model serving instance, you can point to the internal OpenShift service hostname instead of exposing vLLM with a route.
-====
+.Example {rhelai} custom resource file
++
+[source,yaml, subs="attributes,verbatim"]
+----
+apiVersion: ols.openshift.io/v1alpha1
+kind: OLSConfig
+metadata:
+  name: cluster
+  namespace: openshift-lightspeed
+spec:
+  llm:
+    providers:
+    - credentialsSecretRef:
+        name: openai-api-keys
+      models:
+      - name: models/granite-7b-redhat-lab
+      name: rhelai
+      type: rhelai_vllm
+      url: http://3.23.103.8:8000/v1 <1>
+  ols:
+    defaultProvider: rhelai
+    defaultModel: models/granite-7b-redhat-lab
+    logLevel: DEBUG
+----
+<1> The URL endpoint must end with `v1` to be valid.
++
+.Example {rhoai} custom resource file
++
+[source,yaml, subs="attributes,verbatim"]
```

Comment:
Combining the 3 previously-described changes ("CR" + remove continuation + extra space):
```suggestion
.Example {rhoai} CR file
[source,yaml,subs="attributes,verbatim"]
```

Reply:
done